src/WebJobs.Script/Description/ScriptFunctionInvokerBase.cs

```diff
@@ -157,10 +157,12 @@ protected virtual void InitializeEnvironmentVariables(Dictionary<string, string>
             Type triggerParameterType = input.GetType();
             if (triggerParameterType == typeof(HttpRequestMessage))
             {
-                HttpRequestMessage request = (HttpRequestMessage)input;
+                var request = (HttpRequestMessage)input;
+                environmentVariables["REQ_ORIGINAL_URL"] = request.RequestUri.ToString();
```

Comment:
Open to suggestions on the best name here. I also considered REQ_URL, but this name matches what we use in Node (originalUrl).